### PR TITLE
Add support for address on Customer

### PR DESCRIPTION
**Pull request by remi-stripe**
r? @brandur-stripe 
cc @stripe/api-libraries @robertcoe-stripe

### CODE REVIEW

File: invoice.go [hunk 1]
```diff
@@ -193,6 +193,10 @@ type Invoice struct {
 	Currency                  Currency                 `json:"currency"`
 	CustomFields              []*InvoiceCustomField    `json:"custom_fields"`
 	Customer                  *Customer                `json:"customer"`
+	CustomerAddress           *Address                 `json:"customer_address"`
+	CustomerName              *Address                 `json:"customer_name"`
```
Comment (brandur-stripe) [inline invoice.go]: I think this was meant to be a `string` along with `CustomerPhone` below.

**Comment by remi-stripe** [comment]
@brandur-stripe Nice catch! Fixed, PTAL!

File: customer.go [hunk 2]
```diff
@@ -25,23 +25,27 @@ const (
...
+	DefaultSource      *string                        `form:"default_source"`
+	Description        *string                        `form:"description"`
+	Email              *string                        `form:"email"`
+	InvoicePrefix      *string                        `form:"invoice_prefix"`
+	InvoiceSettings    *CustomerInvoiceSettingsParams `form:"invoice_settings"`
+	Name               *string                        `form:"name"`
+	PaymentMethod      *string                        `form:"payment_method"`
+	Phone              *string                        `form:"phone"`
+	Plan               *string                        `form:"plan"`
+	PreferredLanguages []*string                      `form:"preferred_languages"`
```
Comment (mickjermsurawong-stripe) [inline customer.go]: ah this should be "preferred_locales"?
Same as below

**Comment by mickjermsurawong-stripe** [comment]
ptal @remi-stripe

**Comment by remi-stripe** [comment]
@mickjermsurawong-stripe Fixed, PTAL

**Comment by mickjermsurawong-stripe** [comment]
LGTM
thank you @remi-stripe!

**Pull request merged**